src/background/vmModules/ModuleManager.ts

```diff
@@ -104,9 +111,9 @@ export class ModuleManager {
     return this.loadModule(network.caipId, method);
   }
 
-  async #getModule(chainId: string): Promise<Module | undefined> {
-    const [namespace] = chainId.split(':');
-
+  async #getModule(scope: string): Promise<Module | undefined> {
```

Comment:
It's not always a scope. Seems like it can be chainID as well. We either should prevent that from happening or rename the param to something like `chainIdOrScope`